legacy.py

```diff
@@ -0,0 +1,22 @@
+from flask import url_for
+
+class Page:
+	def __init__(self, name, **kwargs):
+		self.name = name
+		self.kwargs = kwargs
+
+
+LEGACY_PATH_MAP = {
+	'/index.php': Page('index'),
+	'/about.php': Page('about'),
+	'/trends.php': Page('report', report_id='state-of-the-web'),
+	'/interesting.php': Page('report', report_id='state-of-the-web', start='latest')
```

Comment:
Should we also point http://httparchive.org/downloads.php to some docs on GitHub, or some such? We can drop the MySQL bits, but it'd be nice to preserve guidance on how to access raw archives and BigQuery data, wdyt?

Reply:
Should we redirect it to https://beta.httparchive.org/faq#how-do-i-use-bigquery-to-write-custom-queries-over-the-data ?

Reply:
Yeah, that looks good.